src/types/student-import/student-import.ts

```diff
@@ -0,0 +1,8 @@
+export interface StudentImportInterface {
+    email: string | object;
```

Comment:
Same as above needed object for relation.

Reply:
Leave it. Test after merge to develop.